url.bs

```diff
@@ -1068,6 +1068,34 @@ code point is U+003A (:).
 <p class="note">As per the <a href=#url-writing>URL writing</a> section, only a
 <a>normalized Windows drive letter</a> is conforming.
 
+<p>A string
+<dfn lt="start with a Windows drive letter|starts with a Windows drive letter">starts with a Windows drive letter</dfn>
+if all of the following are true:
+
+<ul class=brief>
+ <li>its <a for=string>length</a> is greater than or equal to 2
+ <li>its first two code points are a <a>Windows drive letter</a>
+ <li>its <a for=string>length</a> is 2 or the third code point is U+002F (/), U+005C (\),
+ U+003F (?), or U+0023 (#).
+</ul>
+
+<div class=example id=example-start-with-a-widows-drive-letter>
+ <table>
+  <tr>
+   <th>String
+   <th>Starts with a Windows drive letter
+  <tr>
+   <td>"<code>c:</code>"
+   <td>✅
+  <tr>
+   <td>"<code>c:/</code>"
+   <td>✅
+  <tr>
+   <td>"<code>c:a</code>"
+   <td>❌
+ </table>
+</div>
+
```

Comment:
I'm not a huge fan of using symbols to represent the values of the boolean.

Reply:
Which one is preferred: `yes`/`no` or `true`/`false`?

cc @annevk